consistence-coding-standard.md

```diff
@@ -76,6 +76,18 @@ PHP files
 * Long opening tags are used (always `<?php`, never `<?`).
 * There is one empty line after the line with the open tag.
 * File either declares new symbols (classes, functions, constants, etc.) and causes no other side effects, or executes logic with side effects, but should not do both.
+* Uses strict typing by enabling `declare(strict_types = 1);`.
+  * This declaration is placed on a separate line after the opening tag and before other content of the file.
```

Comment:
nějak bych do toho rovnou napsal, že je tam ten volný řádek (takhle to vypadá, že je to hned na dalším, než se dočtu o 2 odrážky níž)